WalletWasabi.Gui/MainWindow.xaml.cs

```diff
@@ -104,7 +105,9 @@ private async Task ClosingAsync()
 						{
 							Global.UiConfig.WindowState = WindowState;
 
-							Global.UiConfig.LastActiveTab = IoC.Get<IShell>().SelectedDocument?.GetType().Name;
+							IDocumentTabViewModel? selectedDocument = IoC.Get<IShell>().SelectedDocument;
+							Global.UiConfig.LastActiveTab = selectedDocument is { } ? selectedDocument.GetType().Name : "";
```

Comment:
```
				WasabiDocumentTabViewModel tabToOpen = UiConfig.LastActiveTab switch
				{
					nameof(SendTabViewModel) => SendTab,
					nameof(ReceiveTabViewModel) => ReceiveTab,
					nameof(CoinJoinTabViewModel) => CoinjoinTab,
					nameof(BuildTabViewModel) => BuildTab,
					_ => HistoryTab
				};
```

So when you assign `Global.UiConfig.LastActiveTab = ""`, you just say "Show history tab"? Maybe then it would be nicer to assign `nameof(HistoryTabViewModel)`. 

Feel free to ignore if I didn't get it right.

Reply:
Done.